### PR TITLE
feat: `LeastActiveRequest` (and `LeastActiveConnection`) supports setting the bias of active request metric

### PR DESCRIPTION
**Pull request by jizhuozhi**
### Issues associated with this PR

#2285 

### Solutions

Keep consistency between mosn and envoy

### UT result
Unit Test is needed if the code is changed, your unit test should cover boundary cases, corner cases, and some exceptional cases. And you need to show the UT result.

### Benchmark
If your code involves the processing of every request, you should give the Benchmark Result.

### Code Style
+ Make sure `Goimports` has run
+ Show `Golint` result

### CODE REVIEW

File: pkg/config/v2/loadbalancer.go [hunk 1]
```diff
@@ -22,7 +22,8 @@ import (
 )
 
 type LeastRequestLbConfig struct {
-	ChoiceCount uint32
+	ChoiceCount       uint32
+	ActiveRequestBias float64 `json:"active_request_bias,omitempty"`
```
Comment (doujiang24) [inline pkg/config/v2/loadbalancer.go:26]: we'd better write some comments for these configurations, otherwise, LGTM.

Comment (jizhuozhi) [reply pkg/config/v2/loadbalancer.go:31]: Resolved

**Comment by jizhuozhi** [comment]
Hello, @doujiang24 , @taoyuanyuan and @alpha-baby , PLAT, THX~

**Review by doujiang24 — APPROVED** [review]

File: pkg/upstream/cluster/lb_leastconnection.go [hunk 2]
```diff
@@ -18,20 +18,23 @@
 package cluster
 
 import (
+	"math"
```
Comment (spacewander) [inline pkg/upstream/cluster/lb_leastconnection.go:21]: Let's format the code with goimports

Comment (jizhuozhi) [reply pkg/upstream/cluster/lb_leastconnection.go:21]: resolved

**Review by doujiang24 — APPROVED** [review]

**Review by doujiang24 — APPROVED** [review]

**Review by spacewander — APPROVED** [review]

**Pull request merged**